src/OptionSettlementEngine.sol

```diff
@@ -42,7 +42,7 @@ contract OptionSettlementEngine is ERC1155, IOptionSettlementEngine {
     mapping(address => uint256) public feeBalance;
 
     /// @notice The address fees accrue to
-    address public feeTo = 0x2dbd50A4Ef9B172698596217b7DB0163D3607b41;
+    address public feeTo;
```

Comment:
The issue with moving this to a constructor arg, is that somebody can snipe the deploy. I think that's why I had it in the bytecode.